tests/app/Http/Controllers/Admin/NotificationAdminControllerTest.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\BaseApiTestCase;
+use App\Models\Controller\ControllerPosition;
+use App\Models\Notification\Notification;
+use App\Providers\AuthServiceProvider;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class NotificationAdminControllerTest extends BaseApiTestCase
+{
+    use DatabaseTransactions;
+
+	protected static $tokenScope = [
+		AuthServiceProvider::SCOPE_DATA_ADMIN
+	];
+
+	protected string $baseEndpoint = "admin/notifications";
+
+	public function testActiveNotificationsCanBeRetrievedByDefault()
+	{
+		// create active notifications
+		Notification::factory()->create();
```

Comment:
Might just be GH  - looks like the formatting / indentation is a bit off here?

Reply:
fixed